Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -401,7 +408,11 @@ extension HabitTrackingController: UITableViewDelegate, UITableViewDataSource {
         
         title.snp.makeConstraints { make in
             make.leading.equalToSuperview().offset(21)
-            make.bottom.equalToSuperview()
+            if section == 0{
+                make.centerY.equalToSuperview()
+            } else {
+                make.bottom.equalToSuperview()
+            }
             make.trailing.equalToSuperview()
             make.height.equalTo(18)
```

Comment:
all numbers in `setupConstraints()` should be constants in the beginning of the function